src/pages/[[...slug]].tsx

```diff
@@ -33,19 +25,33 @@ import {
 } from '@/lib/drupal/staticProps'
 import { StaticPropsResource } from '@/lib/drupal/staticProps'
 import { FormattedPageResource } from '@/data/queries'
-import { LayoutProps } from '@/templates/layouts/wrapper'
+
```

Comment:
I added the checklist imports and alphabetized the rest of this file for easier consumption